### PR TITLE
reorganize classes / interfaces / enums

### PR DESCRIPTION
**Pull request by Z1Meme**
…y command; renamed dynamicWeather.ts to weathermanager.ts; converted dynamic weather worker to a class (WeatherManager); created folders 'managers' and 'models' to reorganize classes, enums, interfaces, etc

### CODE REVIEW

**Comment by Z1Meme** [comment]
TODO: Combine vehicle "passengers" and "seats" properties so they can't get un-synced

**Comment by Z1Meme** [comment]
This is ready but don't delete the branch @QuentinGruber

File: src/servers/ZoneServer2016/commands/commands.ts [hunk 1]
```diff
@@ -1116,7 +1120,11 @@ export const commands: Array<Command> = [
       server.sendGlobalChatText(
         `${targetClient.character.name} has been slain`
       );
-      server.killCharacter(targetClient);
+      const damageInfo: DamageInfo = {
+        entity: client.character.characterId,
+        damage: 999999999,
```
Comment (QuentinGruber) [inline src/servers/ZoneServer2016/commands/commands.ts:1125]: We could add a constant called MAX_UINT32

**Review by QuentinGruber — APPROVED** [review]
🚀

**Pull request merged**